content/fr/learn/research/state-of-innersource-survey-2021.md

```diff
@@ -3,28 +3,25 @@ type: research
 title: 'State of InnerSource Survey 2021'
 image: "/images/learn/research/sois2021.png"
 weight: 4
-summary: Report of the 2021 State of InnerSource Survey
+summary: Rapport de 2021 du State of InnerSource Survey
 research_url: /documents/surveys/State.of.InnerSource.Report.2021.pdf
 survey_closing_date: 2022-01-12
 report_publish_date: 2022-03-22
 ---
 
-We've had a great response from our community and collected over 120 answers to our State of InnerSource 2021 survey. 
-
-This research represents a snapshot of how the InnerSource Commons community is approaching
-InnerSource as a strategy, and where they are on their InnerSource journey in terms of adoption
-status and team readiness. InnerSource culture and an overview of potential blockers and obstacles
-are examined. It also covers the InnerSource practices in most common use, benefits experienced,
-success measures and metrics, and the motivations of those involved. It represents a diverse set of
-experiences across multiple role and organizational profiles from across the globe and we hope it
-provides some inspiration and context to those of you on your own InnerSource journey.
-
-A huge thank you goes to all those who responded to the survey. We encourage all who read this
-report and learn gain insights into what other organizations are doing on their journey to
-InnerSource adoption. If you want to discuss the findings further, do come join us in the
-[InnerSource Commons Slack](/slack); it’s the perfect place to share and learn
-from others who are leading InnerSource practices worldwide.
+Nous avons eu une excellente participation de notre communauté et avons recueilli plus de 120 réponses à notre enquête sur l'état de l'InnerSource 2021. 
 
+Cette recherche représente une vue instantanée de la façon dont la communauté de l'InnerSource Commons aborde la stratégie InnerSource et où elle en est en terme d'adoption, de stratégie, et de maturité des équipes.
+La culture de l'InnerSource est abordé avec ses bloqueurs, et les obstacles potentiels sont analysés.
```

Comment:
abordée

Reply:
Yes,  taken into account in next commit.